flexmeasures/ui/templates/crud/account.html

```diff
@@ -1,183 +1,394 @@
+          >
+            <thead>
+              <tr>
+                <th><i class="left-icon">Name</i></th>
+                <th>Location</th>
+                <th>Asset ID</th>
+                <th>Account</th>
+                <th>Sensors</th>
+                <th class="no-sort">Status</th>
+                <th class="text-right no-sort">
+                  {% if user_can_create_assets %}
+                  <form action="/assets/new" method="get">
+                    <button
+                      class="btn btn-sm btn-responsive btn-success create-button"
+                      type="submit"
+                    >
+                      Create new asset
+                    </button>
+                  </form>
+                  {% endif %}
+                </th>
+                <th class="d-none">URL</th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for asset in assets %}
+              <tr>
+                <td>
+                  <i
+                    class="{{ asset.generic_asset_type.name | asset_icon }} left-icon"
+                    >{{ asset.name }}</i
+                  >
+                </td>
+                <td>
+                  {% if asset.latitude and asset.longitude %} LAT: {{
+                  "{:,.4f}".format( asset.latitude ) }} LONG: {{
+                  "{:,.4f}".format( asset.longitude ) }} {% endif %}
+                </td>
+                <td>{{ asset.id }}</td>
+                <td>
+                  {% if asset.owner %} {{ asset.owner.name }} {% else %} PUBLIC
+                  {% endif %}
+                </td>
+                <td>{{ asset.sensors | length }}</td>
+                <td>
+                  <a href="/assets/{{ asset.id }}/status">
+                    <button type="button" class="btn">Status</button>
+                  </a>
+                </td>
+                <td class="text-right"></td>
+                <td class="d-none">/assets/{{ asset.id }}</td>
+              </tr>
+              {% endfor %}
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-2"></div>
+  </div>
 </div>
-<div class="col-md-2"></div>
 {% block paginate_tables_script %} {{ super() }} {% endblock %}
+<script type="text/javascript">
+  const basePath = window.location.origin;
+  const form = document.getElementById("editaccount");
+  const toastElement = document.getElementById("toastcontainer");
+  const toastDescription = document.getElementById("toastbody");
+
+  function showToast(message, type) {
+    var toastElList = [].slice.call(document.querySelectorAll(".toast"));
+    var toastList = toastElList.map(function (toastEl) {
+      return new bootstrap.Toast(toastEl);
+    });
+    toastList.forEach((toast) => toast.show());
+    toastDescription.innerHTML = message;
+    if (type == "error") {
+      toastElement.classList.add("text-bg-danger");
+    } else {
+      toastElement.classList.add("text-bg-success");
```

Comment:
This is not working, it shows in red when I made a mistake previously (because we are not replacing the other class).

It would be solved if we actually add elements here instead of changing the `toastBody` element. Basically implementing [toast stacking](https://getbootstrap.com/docs/5.0/components/toasts/#stacking) - you already made a container.